react/Modal/index.jsx

```diff
@@ -436,6 +436,8 @@ ModalContent.propTypes = {
 
 const EnhancedModal = migrateProps([
   { src: 'withCross', dest: 'closable' }, // withCross -> closable
+  { src: 'crossClassName', dest: 'closeBtnClassName' }, // withCross -> closable
+  { src: 'crossColor', dest: 'closeBtnColor' }, // withCross -> closable
```

Comment:
Wrong comment due to copy/paste.

Reply:
HAHA I did that yesterday but forgot to push :o